modules/lib/lib-content/src/test/resources/test/QueryContentHandlerTest_dsl_query.js

```diff
@@ -0,0 +1,299 @@
+            'modifier': 'user:system:admin',
+            'createdTime': '1970-01-01T00:00:00Z',
+            'modifiedTime': '1970-01-01T00:00:00Z',
+            'type': 'base:unstructured',
+            'displayName': 'My Content 1',
+            'hasChildren': false,
+            'valid': false,
+            'data': {},
+            'x': {},
+            'page': {},
+            'attachments': {},
+            'publish': {},
+            'workflow': {
+                'state': 'READY',
+                'checks': {}
+            }
+        },
+        {
+            '_id': 'id2',
+            '_name': 'name2',
+            '_path': '/a/b/name2',
+            'creator': 'user:system:admin',
+            'modifier': 'user:system:admin',
+            'createdTime': '1970-01-01T00:00:00Z',
+            'modifiedTime': '1970-01-01T00:00:00Z',
+            'type': 'base:unstructured',
+            'displayName': 'My Content 2',
+            'hasChildren': false,
+            'valid': false,
+            'data': {},
+            'x': {},
+            'page': {},
+            'attachments': {},
+            'publish': {},
+            'workflow': {
+                'state': 'READY',
+                'checks': {}
+            }
+        },
+        {
+            '_id': 'id3',
+            '_name': 'name3',
+            '_path': '/a/b/name3',
+            'creator': 'user:system:admin',
+            'modifier': 'user:system:admin',
+            'createdTime': '1970-01-01T00:00:00Z',
+            'modifiedTime': '1970-01-01T00:00:00Z',
+            'type': 'base:unstructured',
+            'displayName': 'My Content 3',
+            'hasChildren': false,
+            'valid': false,
+            'data': {},
+            'x': {},
+            'page': {},
+            'attachments': {},
+            'publish': {},
+            'workflow': {
+                'state': 'READY',
+                'checks': {}
+            }
+        }
+    ],
+    'aggregations': {
+        'genders': {
+            'buckets': [
+                {
+                    'key': 'male',
+                    'docCount': 10
+                },
+                {
+                    'key': 'female',
+                    'docCount': 12
+                }
+            ]
+        },
+        'by_month': {
+            'buckets': [
+                {
+                    'key': '2014-01',
+                    'docCount': 8
+                },
+                {
+                    'key': '2014-02',
+                    'docCount': 10
+                },
+                {
+                    'key': '2014-03',
+                    'docCount': 12
+                }
+            ]
+        },
+        'price_ranges': {
+            'buckets': [
+                {
+                    'key': 'a',
+                    'docCount': 2,
+                    'to': 50
+                },
+                {
+                    'key': 'b',
+                    'docCount': 4,
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8692790)
```suggestion
                    "docCount": 4,
```